hack/patch-istio-crd.sh

```diff
@@ -124,15 +122,15 @@ function set_fields() {
   local crd_file="${2}"
   local value_name="${3}"
 
-  set_values "${proto_file}"
-
```

Comment:
Don't know why this was called. It's already called in the main function, so I don't think it needs to be called for each top-level value that the `set_values` function itself finds.